pallets/loans/src/lib.rs

```diff
@@ -34,9 +34,11 @@ use frame_support::{
 };
 use frame_system::pallet_prelude::*;
 use orml_traits::{MultiCurrency, MultiCurrencyExtended};
+pub use pallet::*;
```

Comment:
Why do we need this?

Reply:
Runtime needs to find the data types defined in this module.